star-rx72n-firmware/src/rx_clock_power_init.c

```
@@ -156,47 +172,99 @@ static rx_err_t internal_clock_init(void)
 
+/**
+ * @brief Verify system clock configuration is correct
+ *
+ * Checks that the clock system is properly configured to 240 MHz and
+ * that required peripheral modules are enabled.
+ *
+ * @return k_rx_ok if system state is valid, error code otherwise
+ */
+static rx_err_t internal_verify_system_state(void)
+{
+  volatile rx_system_regs_t* sys = system_regs();
```

Clang (14.0.6)

[warning] variable 'sys' is not initialized

(cppcoreguidelines-init-variables)